### DISCUSSION TITLE
Disassembly a build

### DISCUSSION THREAD

**Discussion by tsv1991** [inventree/InvenTree#7156] [Q&A] [answered]
We have case for disassembling build. 
Example of this caes: I made a build of PC from a case, a motherboard, a CPU and a RAM. Sales order was cancel and we need to disassembly a PC to parts and return parts to stock and do delete built part from stock.
I can’t find how to do it. Maybe some one have any ideas?

**Comment by SchrodingersGat** [comment, marked as answer]
@tsv1991 currently the best approach would be to look at the "installed items" tab for the resulting assembled item, and manually "uninstall" those items.

Potentally adding a "break apart" option would be worthwhile - and would like up with this existing issue : https://github.com/inventree/InvenTree/issues/1192

**Comment by tsv1991** [comment]
@SchrodingersGat thanks for tour answer. I'll keep an eye on it
